### ISSUE TITLE
Consider replacing Time to Interactive (beta) WPT calc./metric with standards-based TT(F)I?

### ISSUE THREAD

**Issue by stephendonner** [catchpoint/WebPageTest#1236]
Hi Pat -

tl;dr - would you be interested in using Firefox's new (as of version 64+) ```timeToFirstInteractive``` (TTFI) metric[0], and have it replace "First Interactive (beta)" in the Web UI/potentially other, pre-existing timing metrics/XML?

I want to be careful here, as replacing how that's derived should be consistent among browsers, and, upfront, we are missing the following from https://github.com/WPO-Foundation/webpagetest/blob/8f62b827ad0523acf4303e590371e442dfc3956b/docs/Metrics/TimeToInteractive.md#time-to-first-interactive-calculation:
* ```firstMeaningfulPaint``` (FMP) ([bug 1299117](https://bugzilla.mozilla.org/show_bug.cgi?id=1299117))
* we don't expose the "first interactive window" fully to the DOM, yet, but it's included, otherwise-to-spec using the Long Tasks API, in the millisecond-timestamped ```timeToFirstInteractive``` metrics we return.

(Because we're still also missing (needs impl.) the "in-flight network requests" portion of Time To Consistently Interactive, we don't-yet have that, just FYI.)

Cheers!

/cc @soulgalore @davehunt @rwood-moz @digitarald 

[0] https://searchfox.org/mozilla-central/rev/64ef7bc9fb478ba7c292f9fa57813d3fe864201e/dom/performance/PerformanceTiming.h#427
